be/src/vec/jsonb/serialize.cpp

```diff
@@ -98,11 +102,20 @@ void JsonbSerializeUtil::jsonb_to_block(const DataTypeSerDeSPtrs& serdes, const
     }
     if (filled_columns < dst.columns()) {
         // fill missing slot
-        for (auto& column_type_name : dst) {
+        for (int i = 0; i < dst.columns(); ++i) {
+            const auto& column_type_name = dst.get_by_position(i);
             MutableColumnPtr col = column_type_name.column->assume_mutable();
             if (col->size() < num_rows + 1) {
                 DCHECK(col->size() == num_rows);
-                col->insert_default();
+                if (default_values[i].empty()) {
+                    col->insert_default();
+                } else {
+                    // col->insert_default();
+                    ReadBuffer value(
+                            reinterpret_cast<const unsigned char*>(default_values[i].data()),
+                            default_values[i].size());
+                    static_cast<void>(column_type_name.type->from_string(value, col.get()));
```

Comment:
I remember that from_string is deprecated by amorynan.

Reply:
from_string is still used in many codes

Reply:
done